src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -319,7 +319,61 @@
   end
 
   describe '#user_and_role' do
-    skip('Please add some tests')
+    let(:user) { create(:user) }
+    let(:nobody) { create(:user_nobody) }
```

Comment:
@ChrisBr I moved the creation of the factory for the nobody user to a dedicated commit, see: https://github.com/openSUSE/open-build-service/pull/3102/commits/421f84e02eb00102d5ac4295d7f159b5f56e9ad0

But just for this, I don't think it's necessary to open a new PR.